code/API_definitions/edge-discovery-service.yaml

```diff
@@ -11,14 +11,198 @@ info:
   description: |
     # Introduction
     ---
-    Use the Edge Discovery API for
-    discovery of the most optimal edge cloud zone for a specific user as well as specific application instance the user is trying to connect to.
+    The Edge Discovery API is designed to help users discover the most optimal
+    edge cloud zones for their applications. This API allows users to find the
+    best regions and zones for their edge cloud applications, discover optimal
+    edge cloud zones for deployed applications, and find the best application
+    endpoints for clients to connect to.
 
-    ---
+    ## Features
+    - Discover regions and zones to deploy edge cloud applications.
+    - Discover optimal application endpoints for client connections for deployed applications.
+
+    ## Usage
+    Users can utilize this API to enhance the performance and connectivity of
```

Comment:
It is bit confusing when it says the users can use this API to enhance connectivity and performance by selecting suitable edge cloud zone and application endpoints. In my view the app providers may need to use this API to deploy applications while the end users may use it just to discover application endpoints to connect with. 
So in terms of the usage it would be good to describe the scenario where the given intents are to be used by the intended users as per the goals of the API. Thoughts?